Marharyta_Miliuk/Module 3 - Java/Week 2/NewYearPresent/src/by/epam/cdp/java/nypresent/Runner.java

```diff
@@ -0,0 +1,66 @@
+package by.epam.cdp.java.nypresent;
+import java.util.InputMismatchException;
+
+import by.epam.cdp.java.nypresent.beans.Bar;
+import by.epam.cdp.java.nypresent.beans.ChocolateCandy;
+import by.epam.cdp.java.nypresent.beans.Lollypop;
+import by.epam.cdp.java.nypresent.utils.InputOutputStreams;
+
+public class Runner {
+	
+public static int askNumberOfCandies (){//creating method do define number of candies to put into present
+		
+		System.out.println("How many candies do you want?");
+		
+		return InputOutputStreams.scanNumbers();
+	}
+
+	public static void main (String []args){
+		
+	try{
+		
+		Present present = new Present ();
```

Comment:
please think about loop in the program.
for example you can ask user what to do?
1. Add sweet
2. Show present
3. Sort present by name
4. get weight
5. Exit
and for example use switch/case
while user doesn't enter 5 program should work